README.md

```diff
@@ -36,19 +44,163 @@ ti.tidbMapDatabase("tpch")
 
 spark.sql("select count(*) from lineitem").show
 ```
+Result:
+```
++--------+
+|count(1)|
++--------+
+|    6005|
++--------+
+```
+
+### For spark-submit
+
+1. Download or compile tispark [here](https://github.com/pingcap/tispark) and make sure it is visible in your local [maven](maven.apache.org) repository.
+2. In your project pom.xml, include these dependencies:
+```xml
+<dependencies>
+    <dependency>
+        <groupId>com.pingcap.tispark</groupId>
+        <artifactId>tispark</artifactId>
+        <version>${tispark.version}</version>
+    </dependency>
+
+    <dependency>
+        <groupId>org.apache.spark</groupId>
+        <artifactId>spark-core_${scala.version}</artifactId>
+        <version>${spark.version}</version>
+        <scope>provided</scope>
+    </dependency>
+    <dependency>
+        <groupId>org.apache.spark</groupId>
+        <artifactId>spark-catalyst_${scala.version}</artifactId>
+        <version>${spark.version}</version>
+        <scope>provided</scope>
+    </dependency>
+    <dependency>
+        <groupId>org.apache.spark</groupId>
+        <artifactId>spark-sql_${scala.version}</artifactId>
+        <version>${spark.version}</version>
+        <scope>provided</scope>
+    </dependency>
+    <dependency>
+        <groupId>org.apache.spark</groupId>
+        <artifactId>spark-hive_${scala.version}</artifactId>
+        <version>${spark.version}</version>
+        <scope>provided</scope>
+    </dependency>
+    <dependency>
+        <groupId>org.apache.spark</groupId>
+        <artifactId>spark-unsafe_${scala.version}</artifactId>
+        <version>${spark.version}</version>
+        <scope>provided</scope>
+    </dependency>
+</dependencies>
+
+```  
+3. Write your own application code. In this example, we create a Java file ```com.pingcap.spark.App``` like this:
+
+```java
+package com.pingcap.spark;
+
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.SparkSession;
+import org.apache.spark.sql.TiContext;
+
+public class App {
+  public static void main(String[] args) {
+    SparkSession spark = SparkSession
+            .builder()
+            .appName("TiSpark Application")
+            .getOrCreate();
+
+    TiContext ti = new TiContext(spark);
+    ti.tidbMapDatabase("tpch", false);
+    Dataset dataset = spark.sql("select * from customer");
```

Comment:
use select count(*) and reduce output you need to paste here below.

Reply:
done